pkg/util/cluster/cluster.go

```diff
@@ -53,10 +54,9 @@ type Cluster struct {
 	ci           bool
 	ciParentVnet string
 
+	graph                             *msgraph.GraphServiceClient
```

Comment:
nit: `graph` is a little vague unless you look at the type. Clusters also have install graphs, might be better as `msgraph`

Reply:
I agree with that.  I think I called it `fpGraphClient` elsewhere.  I'll make it consistent.

Reply:
Updated the `pkg/util/cluster` commit.